openedx_lti_tool_plugin/deep_linking/api/views.py

```diff
@@ -0,0 +1,54 @@
+"""Django Views."""
+from django.http.request import HttpRequest
+from django.utils.decorators import method_decorator
+from pylti1p3.exception import LtiException
+from rest_framework.exceptions import APIException
+from rest_framework.status import HTTP_400_BAD_REQUEST
+from rest_framework.viewsets import GenericViewSet
+
+from openedx_lti_tool_plugin.deep_linking.exceptions import DeepLinkingException
+from openedx_lti_tool_plugin.deep_linking.utils import validate_deep_linking_message
+from openedx_lti_tool_plugin.mixins import LTIToolMixin
+from openedx_lti_tool_plugin.views import requires_plugin_enabled
+
+
+@method_decorator(requires_plugin_enabled, name='dispatch')
```

Comment:
```suggestion
@method_decorator(requires_plugin_enabled, name='dispatch')
```
@kuipumu The name of this decorator is too general. Which plugin is required to be enabled?